cmd/cosign/cli/verify/verify_blob.go

```diff
@@ -115,29 +126,14 @@ func (c *VerifyBlobCmd) Exec(ctx context.Context, blobRef string) error {
 	if c.RFC3161TimestampPath != "" && c.KeyOpts.TSACertChainPath == "" {
```

Comment:
This check needs to be updated since you can use `UseSignedTimestamps` as well with `RFC3161TimestampPath`